src/core/udb-api.service.js

```diff
@@ -231,6 +231,41 @@ function UdbApi(
     return deferredOffer.promise;
   };
 
+  /**
+   * @param {Array} events
+   * @return {Array}
+   */
+  this.reformatEvents = function(events) {
+    events.member = events.member.map(function(member) {
+      var memberContext = (member['@context']) ? member['@context'].split('/').pop() : '';
+      memberContext = memberContext.charAt(0).toUpperCase() + memberContext.slice(1);
+      member['@type'] = (member['@type']) ? member['@type'] : memberContext;
+      return member;
+    });
+    return events;
+  };
+
+  /**
+   * @param {object} event
+   * @return {object}
+   */
+  this.formatOfferClass = function(event) {
```

Comment:
This can also return an `UdbOrganizer` which is not an `Offer`. `Offer` is an umbrella term for `Event` and `Place` (aka "Aanbod" in Dutch). I'm okay with having one function for all three but it should have a different name. The `event` argument name is also incorrect since it's actually json data that can either be an event, place, or organizer.
I'd suggest `this.formatJsonLDEntity = function(jsonLD) { ... }` but that's just of the top of my head so feel free to come up with other names.